velox/functions/prestosql/WidthBucketArray.cpp

```diff
@@ -22,6 +22,13 @@
 namespace facebook::velox::functions {
 namespace {
 
+// Presto Java seems implicitly treating null elements as zeros.
+// Adhere to this behavior for now.
+template <typename T>
+T valueOrZero(const DecodedVector& elementsHolder, vector_size_t idx) {
+  return elementsHolder.isNullAt(idx) ? T{0} : elementsHolder.valueAt<T>(idx);
```

Comment:
@Yuhta I understand your position and I also want to fix it properly.
However, at the moment it will break some queries and behavior will differ from Presto.
This will complicate the adoption and roll out of the Prestissimo for both users and us.

I have created Presto issue to come with the conclusion of what we want to do about this problem: https://github.com/prestodb/presto/issues/24055

Hopefully, soon we can alter Presto's behavior and with it, the Velox's.

Thanks!

Reply:
The Java queries have NULLs in bins will for sure not giving what the user expects.  If we are worry about user reports, just throw `VELOX_FAIL_UNSUPPORTED_INPUT_UNCATCHABLE` so the queries will be retried on Java.